README.md

```diff
@@ -1,24 +1,18 @@
 <h1 align="center">
-    <img src="/aksel.nav.no/website/public/favicon.svg" style="width:13rem"/>
+    <img src="/aksel.nav.no/website/public/favicon.svg" style="width:13rem" />
     <br/><br/>Aksel
 </h1>
 
 <div align="center">
+    <p>Nav's design system + Aksel.nav.no</p>
     <p>
-        Navs designsystem + Aksel.nav.no
+      <a href="https://aksel.nav.no/storybook/"><img alt="Storybook" src="https://shields.io/badge/storybook-white?logo=storybook&style=flat" /></a>&nbsp;
```

Comment:
Removing the line breaks before and after `<img>` removes the blue underlines:
![image](https://github.com/user-attachments/assets/1dc3ca7d-f2a1-4bcc-8eb4-ce79c8d08c95)

Adding `&nbsp;` just adds some more spacing, but isn't strictly needed.